Mathlib/Topology/DiscreteQuotient.lean

```diff
@@ -0,0 +1,405 @@
+/-
+Copyright (c) 2021 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Calle Sönne, Adam Topaz
+
+! This file was ported from Lean 3 source module topology.discrete_quotient
+! leanprover-community/mathlib commit 0a0ec35061ed9960bf0e7ffb0335f44447b58977
```

Comment:
Let's make sure we remember to update this to reflect the backport before merging

Reply:
I will update SHA sums in 2 files before making it `awaiting-review`.